modules.d/99openssl/module-setup.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+check() {
+    require_binaries openssl || return 1
```

Comment:
The `openssl` binary is typically not installed by default on systems, and it doesn't need to be present for applications to use OpenSSL's libraries.

Chances are there's a non-zero number of deployments out there that will fail this check.